lib/apis/v1/policy.go

```diff
@@ -123,15 +123,15 @@ type PolicySpec struct {
 
 	// Types indicates whether this policy applies to ingress, or to egress, or to both.  When
 	// not explicitly specified (and so the value on creation is empty or nil), Calico defaults
-	// Types according to what IngressRules and EgressRules are present in the policy.  The
+	// Types according to what Ingress and Egress are present in the policy.  The
```

Comment:
this change lines up nicely with k8s - type: `NetworkPolicy[In|E]gressRule` -> `Rule` and now `[In|E]gress` == `[In|E]ngress`.